MobileFilter/sections/specific_web.txt

```diff
@@ -76,6 +76,8 @@ scmp.com##.section-content-bottom__ad-container
 web.de,gmx.*##.base-page__ad
 youmath.it###contenuto > #T[style="text-align: center;"]:empty
 hd21.com##.mercantile_title
+ruliweb.com##.mo_ad_main_ti_wrapper
+||ruliweb.com/js/criteo.publishertag.js
```

Comment:
Is not it a tracker?

Reply:
@Alex-302, It has ad intergration and adblocking code inside, so i added it as a ad script.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/105973339-adfb6d80-609d-11eb-818b-3d8523783edd.png)
</details><br/>

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/105973397-beabe380-609d-11eb-93ec-110c23ab4fe7.png)
</details><br/>